pyseir/load_data.py

```diff
@@ -365,7 +365,7 @@ def load_cdc_hospitalization_data():
     return pd.read_csv(os.path.join(DATA_DIR, "cdc_hospitalization_data.csv"))
 
 
-@lru_cache(maxsize=1)
+@lru_cache(maxsize=None)
 def load_mobility_data_m50():
```

Comment:
Are these load functions used anywhere?

Reply:
looks like only load_public_implementations_data. I'm not going to delete them until they get in the way.